src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -13,6 +13,10 @@ namespace Xamarin.Android.Tasks
 {
 	public class GeneratePackageManagerJava : Task
 	{
+		const string EnvironmentFileName = "XamarinAndroidEnvironmentVariables.java";
```

Comment:
Should this be in `<GeneratePackageManagerJava/>` and not a new Task?  @dellis1972?

Reply:
We discussed that with @dellis1972, this location was his recommendation.